core/config/toml/types.go

```diff
@@ -1553,6 +1555,59 @@ func (t *Tracing) ValidateConfig() (err error) {
 	return err
 }
 
+type Beholder struct {
+	//TODO explicit enable? Or just empty endpoint?
+	CACertFile           *string
+	Endpoint             *string
+	EmitterExportTimeout *commonconfig.Duration
+	InsecureConnection   *bool
+	LogExportTimeout     *commonconfig.Duration
+	MetricReaderInterval *commonconfig.Duration
+	PackageName          *string           //TODO remove?
```

Comment:
What is the intent for this config field? Example values?

Reply:
Ah, I see https://github.com/smartcontractkit/chainlink-common/pull/694#discussion_r1715428798 now